main_test.go

```diff
@@ -77,7 +77,18 @@ func TestLifecycle(t *testing.T) {
 		{
 			name: "url denied",
 			inlineRules: `
-			SecRuleEngine On\nSecRule REQUEST_URI \"@streq /hello?name=panda\" \"id:101,phase:1,t:lowercase,deny\"
+			SecRuleEngine On\nSecRule REQUEST_URI \"@beginsWith /hello?name=panda\" \"id:101,phase:1,t:lowercase,deny\"
```

Comment:
This change is incidental, as after adding XSS, match won't happen so don't pay too much attention to it.